### ISSUE TITLE
Does Trino to Spark only support conversion syntax differences? Does it not support conversion of function differences?

### ISSUE THREAD

**Issue by HelloZZF** [linkedin/coral#413]
I tried to convert these two functions, but both resulted in an error message "No match found for function".
1、select regexp_like('a','\d')
2、select date_diff('month',cast('2022-05-23' as date),cast('2022-06-23' as date))
![733A80B6-9BE2-4829-8747-5B6A6C557A9F](https://github.com/linkedin/coral/assets/31527268/126bf2be-c2c2-457f-9e6f-fcccea7ebf19)

When I try to convert SQL statements with syntax differences, Coral can successfully perform the conversion, for example:
SELECT *
FROM mydb.mytable AS a
CROSS JOIN UNNEST(split(a.id, ',')) AS t0 (b)

May I ask if the conversion of Trino to Spark function differences is not currently supported? 
I see validator. validate (query) in the code
It is verified using the rules of hive, and the unique function in trino will report an error at this step

**Comment by wmoustafa** [comment]
Yes, Trino to Coral is not fully supported yet and it is work in progress.

**Comment by HelloZZF** [comment]
Got it，Thank you for your reply.
